Formula/libmatheval.rb

```diff
@@ -0,0 +1,40 @@
+class Libmatheval < Formula
+  desc "Library to parse and evaluate symbolic expressions input as text."
+  homepage "https://www.gnu.org/software/libmatheval/"
+  url "https://ftpmirror.gnu.org/libmatheval/libmatheval-1.1.11.tar.gz"
+  sha256 "474852d6715ddc3b6969e28de5e1a5fbaff9e8ece6aebb9dc1cc63e9e88e89ab"
+
+  depends_on "bdw-gc" => :linked
+  depends_on "bison" => :build
+  depends_on "guile"
+
+  def install
+    # The configure script adds the Guile link directives in the wrong order.
```

Comment:
This is a brand new formula, so it would be nice if we weren't already having to patch it. Has this been reported upstream?